packages/gatsby-plugin-static-image/src/image-processing.ts

```diff
@@ -34,9 +35,21 @@ export async function writeImages({
       const filename = path.join(cacheDir, `${hash}.json`)
       try {
         const options = { file, args, reporter, cache }
-        const data = await (isFixed ? fixedSharp(options) : fluidSharp(options))
+        // get standard set of fields from sharp
+        const sharpData = await (isFixed
+          ? fixedSharp(options)
+          : fluidSharp(options))
+        if (sharpData) {
+          // get the equivalent webP and tracedSVG fields gatsby-transformer-sharp handles normally
+          const customSharpFields = await getCustomSharpFields({
+            isFixed: isFixed ? true : false,
```

Comment:
Rather than this, let's cast `isFixed` to boolean when we create it:

`const isFixed = Boolean(fixed ?? !fluid)`